python/rpdk/java/__init__.py

```diff
@@ -1,5 +1,5 @@
 import logging
 
-__version__ = "0.1.2"
+__version__ = "0.1.3"
```

Comment:
why is this needed if nothing on the python side is changing? should we just get in the habit of bumping both any time we make a change?

Reply:
the template was changed, innit?

Reply:
ah 🤦‍♂️ my bad